docs/clean/user_guide/executing_a_clean.rst

```diff
@@ -47,11 +47,20 @@ Passing images through CLI
     .. code-block:: bash
 
         # Example of passing an image to a device called 'PE1'
-        pyats clean --testbed-file </path/to/testbed.yaml> --clean-file </path/to/clean.yaml> --clean-image PE1:</path/to/image.bin>
+        pyats clean --clean-image PE1:</path/to/image.bin> --testbed-file </path/to/testbed.yaml> --clean-file </path/to/clean.yaml>
 
         # Example of passing an image to all devices with the 'n9k' platform
-        pyats clean --testbed-file </path/to/testbed.yaml> --clean-file </path/to/clean.yaml> --clean-platform n9k:</path/to/image.bin>
+        pyats clean --clean-platform n9k:</path/to/image.bin> --testbed-file </path/to/testbed.yaml> --clean-file </path/to/clean.yaml>
```

Comment:
Just moved the --clean-image and --clean-platform arguments to the front for better visibility